packages/flutter/lib/src/material/theme_data.dart

```diff
@@ -1815,6 +1815,15 @@ class VisualDensity extends Diagnosticable {
     );
   }
 
+  /// Calculate effective constraints for a widget with constraints [constraints].
```

Comment:
This doesn't really explain what the method does.

Return a copy of [constraints] whose minimum width and height have been updated with the [baseSizeAdjustment].

Reply:
Edited. Thank you!